### DISCUSSION TITLE
read more about BPF usage with memcached

### DISCUSSION THREAD

**Discussion by osevan** [dragonflydb/dragonfly#131] [Q&A] [answered]
Here you can read about usage :

Maybe you can do this for DF too than you have more speed for redis and memcached api. 

https://github.com/Orange-OpenSource/bmc-cache

**Comment by romange** [comment, marked as answer]
While this looks super interesting, it won't get lots of traction with mainstream use-cases because BPF usually handles UDP protocol.. 

Usually, application-level clients use a more reliable and simple to operate TCP protocol. DF bridges the gap between mainstream and performance. We won't be able to pursue extreme directions that require a very special setup or limited protocols.
